code/modules/projectiles/projectile.dm

```diff
@@ -111,6 +111,7 @@
 		return
 	if(AM.projectile_hit(src))
 		AM.do_projectile_hit(src)
```

Comment:
doenst this call qdel?

Reply:
nop, it just applies damage and tuti quanti